docs/reference/data-streams/data-streams.asciidoc

```diff
@@ -47,16 +46,152 @@ We recommend using data streams if you:
 
 
 [discrete]
-[[data-streams-toc]]
-== In this section
+[[backing-indices]]
+== Backing indices
+A data stream consists of one or more _backing indices_. Backing indices are
+<<index-hidden,hidden>>, auto-generated indices used to store a stream's
+documents.
 
-* <<data-streams-overview>>
-* <<set-up-a-data-stream>>
-* <<use-a-data-stream>>
-* <<data-streams-change-mappings-and-settings>>
+image::images/data-streams/data-streams-diagram.svg[align="center"]
 
+To create backing indices, each data stream requires a matching
+<<indices-templates,index template>>. This template acts as a blueprint for the
+stream's backing indices. It contains:
+
+* The mappings and settings applied to each backing index when it's created.
+
+* A name or wildcard (`*`) pattern that matches the data stream's name.
+
+* A `data_stream` object with an empty body (`{ }`). This object indicates the
+template is used for data streams.
+
+A `@timestamp` field must be included in every document indexed to the data
+stream. This field can be mapped as a <<date,`date`>> or
+<<date_nanos,`date_nanos`>> field data type in the stream's matching index
+template. If no mapping is specified in the template, the `date` field data type
+with default options is used.
+
+The same index template can be used to create multiple data streams.
+
+
+[discrete]
+[[data-streams-generation]]
+== Generation
```

Comment:
Maybe Data stream generations? On it's own, it brings to mind generating data streams.